pcs/lib/pacemaker/values.py

```diff
@@ -61,6 +67,18 @@ def is_score(value: str) -> bool:
     return unsigned_value == SCORE_INFINITY or unsigned_value.isdigit()
 
 
+def is_duration(runner: CommandRunner, value: str) -> bool:
```

Comment:
Please add test coverage for this function into file [pcs_test/tier0/lib/pacemaker/test_values.py](https://github.com/ClusterLabs/pcs/blob/main/pcs_test/tier0/lib/pacemaker/test_values.py). That would be two simple tests with mocked runner.